checkio_sendgrid/settings.py

```diff
@@ -1,2 +1,4 @@
-DEFAULT_TIMEOUT = 10
-SENDGRID_USER_ADD_LIMIT = 1000
+from django.conf import settings
+
+DEFAULT_TIMEOUT = getattr(settings, 'DEFAULT_TIMEOUT', 10)
```

Comment:
in project settings, all setting items must have prefix:
DEFAULT_TIMEOUT = getattr(settings, 'SENDGRID_DEFAULT_TIMEOUT', 10)